Riot/Modules/MatrixKit/Models/Account/MXKAccount.m

```diff
@@ -626,6 +673,60 @@ - (void)load3PIDs:(void (^)(void))success failure:(void (^)(NSError *))failure
     }];
 }
 
+- (void)loadCurrentPusher:(void (^)(void))success failure:(void (^)(NSError *error))failure
+{
+    if (!self.mxSession.myDeviceId)
+    {
+        return;
```

Comment:
Does it make sense to call the failure block here?

Reply:
Good catch! Thank you :)